### PR TITLE
Convert to CPS

### PR DESCRIPTION
**Pull request by RussKie**
* Retarget to .NET Standard 2.0
* Fix compile issues
* Fix warnings

### CODE REVIEW

**Comment by RussKie** [comment]
@pmiossec I think you've been dealing with the GH plugin. 
Do you think you could check that the update still works?

**Review by gerhardol — COMMENTED** [review]
No important comment. Leaner definition, the plugin works for me.
What about the latest changes in master?

Got an exception when opening after building, not sure if it is related. Not occurring after a rebuild.
<details>

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at GitHub3.GitHub3Plugin.GetHostedRemotesForModule() in F:\dev\gc\gitextensions_4\Plugins\GitHub3\GitHub3Plugin.cs:line 190
   at GitHub3.GitHub3Plugin.GitModuleIsRelevantToMe() in F:\dev\gc\gitextensions_4\Plugins\GitHub3\GitHub3Plugin.cs:line 182
   at GitUI.PluginRegistry.<>c.<TryGetGitHosterForModule>b__11_0(IRepositoryHostPlugin gitHoster) in F:\dev\gc\gitextensions_4\GitUI\Plugin\PluginRegistry.cs:line 63
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at GitUI.PluginRegistry.TryGetGitHosterForModule(GitModule module) in F:\dev\gc\gitextensions_4\GitUI\Plugin\PluginRegistry.cs:line 63
   at GitUI.CommandsDialogs.FormBrowse.TryGetRepositoryHost(IRepositoryHostPlugin& repoHost) in F:\dev\gc\gitextensions_4\GitUI\CommandsDialogs\FormBrowse.cs:line 2138
   at GitUI.CommandsDialogs.FormBrowse._viewPullRequestsToolStripMenuItem_Click(Object sender, EventArgs e) in F:\dev\gc\gitextensions_4\GitUI\CommandsDialogs\FormBrowse.cs:line 2104
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
</details>

**Comment by RussKie** [comment]
> Got an exception when opening after building, not sure if it is related. Not occurring after a rebuild.

Thank you for testing.
Not sure, could be a red herring. It worked for me on two different computers.

**Review by drewnoakes — APPROVED** [review]

File: Directory.Build.props [hunk 1]
```diff
@@ -1,3 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+
+  <PropertyGroup>
+    <Copyright>Copyright &#xA9; 2012 Marcus Bauer &lt;mabako@gmail.com&gt; // Copyright &#xA9; 2019 Git Extensions</Copyright>
```
Comment (pmiossec) [inline Directory.Build.props:5]: Is there a particular reason to put it here instead in the **only** csproj the repository have?

**Review by pmiossec — APPROVED** [review]
It is working well for me.

**Comment by pmiossec** [comment]
For the other csproj, we could do it with https://github.com/hvanbakel/CsprojToVs2017
Tested successfully. It does most of the job...

File: Directory.Build.props (hunk 1 above)
Comment (RussKie) [reply Directory.Build.props:5]: In the target state we'll put all the common info into a directory.build.props
However right now I don't want to deal with this.

**Comment by RussKie** [comment]
> For the other csproj, we could do it with [hvanbakel/CsprojToVs2017](https://github.com/hvanbakel/CsprojToVs2017)

Nice! Do you want to try it on other submodules?

**Pull request merged**